### ISSUE TITLE
migrating from pycrypto to pycryptodome

### ISSUE THREAD

**Issue by noraj** [ShawnDEvans/smbmap#31]
Please consider migrating from pycrypto to pycryptodome:

- [Stop using pycrypto. Use pycryptodome instead](https://blog.sqreen.com/stop-using-pycrypto-use-pycryptodome/)
- Official: https://github.com/dlitz/pycrypto/issues/238
- Similar: https://github.com/oduwsdl/ipwb/issues/303

> 
![image](https://user-images.githubusercontent.com/16578570/57192923-29b29f80-6f36-11e9-9965-68d54599aef8.png)

**Comment by Edu4rdSHL** [comment]
Hello, @noraj as I said in https://github.com/BlackArch/blackarch/issues/2325#issuecomment-492491218 it isn't an issue with smbmap, it's with the impacket package of ArchLinux. You can close it.

**Comment by noraj** [comment]
@Edu4rdSHL Yeah but even if the problem doesn't from upstream, it is a bad thing to require pycrypto and better to require pycryptodome or pycryptodomex instead.

**Comment by MarioVilas** [comment]
Latest Impacket is using pycryptodomex, this issue can be closed.
